### ISSUE TITLE
[windows] please add abilities to edit already scheduled task and to exclude subdirs

### ISSUE THREAD

**Issue by ache** [conformal/cyphertite#72]
Hi.
1) The only way to change scheduled task is to re-create it. It is bothering, if many directories are handled. Editing scheduled time will be also handy.
2) There is no way to exclude some subdirs from the GUI and to run CLI for that is not convenient. Please add such ability.

**Comment by csmiken** [comment]
We are working on this feature now. It should be added in the next Windows release. The ETA is soon-ish? But i cannot promise when at this time.

**Comment by ache** [comment]
Thanx. There are two more things left to make schedule under windows useable. 
When scheduled time is missed (f.e. due to computer power off) scheduled task does not run immediately when computer starts. Please note that Windows machine is not 24/7 Unix server and missed schedule times is normal situation, so a checkbox need to be added to schedule properties to run it as soon as possible, there is a chance to never run it otherwise.
Next thing is some sort of Disable (not Remove) checkbox to temporary not run scheduled task at its given time (due to planned activity with backup files, etc.)

**Comment by csmiken** [comment]
Also great Ideas! When I Was talking with my developers about this, we worked on all of the above. My wife is asking for the same features so hopefully they will make it into the next release also. We are actively looking into all of the things you have mentioned... We just need to get the code written and tested.

**Comment by csmiken** [comment]
We have added the ability to edit the file paths, tag name, and schedule frequency/time of an existing scheduled backup to Cyphertite 2.0.1.  We have also added a "Run Now" button to immediately run scheduled backups.  While still not completely perfect, these additions should make the scenarios above much easier to perform.

As far as excluding subdirectories, we are keeping #63 open for that, and may add this functionality in the future.
